taxi/views.py

```diff
@@ -61,16 +63,47 @@ class CarListView(LoginRequiredMixin, generic.ListView):
 class CarDetailView(LoginRequiredMixin, generic.DetailView):
     model = Car
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        car = self.object
+        user = self.request.user
+
+        if isinstance(user, Driver):
```

Comment:
why do you need this check?
you have LoginRequiredMixin, thus only authenticated users can acces this view, not anonymous
your AUTH_USER_MODEL = Driver, thus request.user is always Driver or I'm missing something?
Fix everywhere